backend/api-request.http

```diff
@@ -0,0 +1,34 @@
+# vscode extension "Humao Rest Client"
+
+@baseUrl = http://urbanplants.local
+
+###
+
+GET {{baseUrl}}/plant
+
+###
+
+GET {{baseUrl}}/plant/1
+
+###
+
+DELETE {{baseUrl}}/plant/12
+
+###
+
+PUT {{baseUrl}}/plant/5
+
+  {
+    "plantname": "Hello and thx for reviewing this important part of my capstone project :D",
+    "plantspecies": "Let me tell you a story, how I lost 1 hour of my time, just because I had a , on the wrong place of my test-json down below."
```

Comment:
Business as usual ;-)
Googling a day and then deciding to try a complete different approach...